.github/workflows/release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Build & Run Development Container
         uses: devcontainers/ci@v0.2
         with:
-          imageName: ghcr.io/customink/crypteia
+          imageName: ghcr.io/customink/crypteia-devcontainer
```

Comment:
Not part of the PR here but I wanted to have each package (two container images soon) associated with this repo have intention revealing names.